bqetl_project.yaml

```diff
@@ -385,6 +385,18 @@ generate:
   stable_views:
     skip_datasets:
     - mlhackweek_search
+  usage_reporting:
+    apps:
+      fenix:
+        channels:
+        - nightly
+      # firefox_ios:
+      #   channels:
+      #   - nightly
```

Comment:
I checked, we should have data there now https://sql.telemetry.mozilla.org/queries/105052/source

Reply:
I will take a look later. Thanks for letting me know.